Source/Atomic/UI/UIWidget.h

```diff
@@ -250,10 +258,18 @@ class UIWidget : public Object, public tb::TBWidgetDelegate
     bool GetCaptured();
 
     void SetCapturing(bool capturing);
```

Comment:
Ok, we'll try and figure out what to call this later, and at some point maybe there will be a slightly way of handling it anyway, so in the future I'll flag method/var names which may cause some confusion, better :)